feature-runner/lib/codeBlockReplacer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { codeBlockReplacer } from './codeBlockReplacer.js'
```

Comment:
Replacing data from context is already implemented in BDD Markdown. Use that method.